CIP-?/README.md

```diff
@@ -0,0 +1,297 @@
+---
+CIP: ?
+Title: Governance Action Metadata Standard
```

Comment:
```suggestion
Title: Governance Metadata Standard
```
Match the title on the PR? makes this _less_ 1694 specific?

Reply:
@Ryun1 @Quantumplation from a first reading it does seem like this CIP creates a general vocabulary around governance... not just governance action... so I would also be in favour of this change.

Reply:
The main awkwardness is I don't have a good word for "any action taken pertaining to governance"; When I wrote this, what CIP-1694 refers to as "Governance Actions", I thought of as "Governance Proposals".

I've kind of taken to using the term "governance transactions", but I'm open to other suggestions too.